resources/helm/v2.1/istio-init/files/servicemeshextensions.maistra.io.crd.yaml

```diff
@@ -1,50 +1,53 @@
-
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   labels:
     maistra-version: "2.1.0"
-    chart: istio
-    release: istio
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.0
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
Hopefully with the move to maistra/api we are having only one version. Currently operator is 0.4 and istio is 0.2.4